grpc-server/template_test.go

```diff
@@ -22,17 +25,31 @@ tasks:
       timeout: 60`
 
 	template2 = `version: "0.1"
-name: hello_world_workflow
+name: hello_world_again_workflow
 global_timeout: 600
 tasks:
   - name: "hello world again"
-    worker: "{{.device_1}}"
+    worker: "{{.device_2}}"
     actions:
-  	- name: "hello_world_again"
+    - name: "hello_world_again"
       image: hello-world
       timeout: 60`
 )
 
+type templates struct {
+	id   uuid.UUID
+	data string
+}
+
+var templateDB = map[string]interface{}{}
+
+// ClearTemplateDB clear all the templates
+func clearTemplateDB() {
+	for name := range templateDB {
+		delete(templateDB, name)
+	}
```

Comment:
can't we just do `templateDB = map[string]interface{}{}`?

Reply:
Yes, we can do that. Thanks

Reply:
Done